### PR TITLE
PR to test Driver S3 upload

### PR DESCRIPTION
**Pull request by jonahjon**
Signed-off-by: jonahjon <jonahjones094@gmail.com>

### CODE REVIEW

File: config/jobs/build-drivers/build-drivers.yaml [hunk 1]
```diff
@@ -195,27 +197,29 @@ periodics:
...
-        image: 292999226676.dkr.ecr.eu-west-1.amazonaws.com/test-infra/build-drivers:latest
+        image: 292999226676.dkr.ecr.eu-west-1.amazonaws.com/test-infra/build-drivers:s3-publish
         imagePullPolicy: Always
         securityContext:
           privileged: true
   - name: build-drivers-debian-periodic
     decorate: true
-    cron: "0 03 * * 4"
+    interval: 1h
+    # cron: "0 03 * * 4"
```
Comment (leodido) [inline config/jobs/build-drivers/build-drivers.yaml:207]: Ok, this periodic `ProwJob` seems to have worked 👉  https://download.falco.org/?prefix=driver/5c0b863ddade7a45568c0ac97d037422c9efb750/

Anyways, I was expecting the presubmit build jobs to run on this PR...

**Review by leodido — COMMENTED** [review]
Thanks for approaching this Jonah!

It's definitely in the right direction. 🥳 

Left a comment.

File: config/jobs/build-drivers/build-drivers.yaml (hunk 1 above)
Comment (jonahjon) [reply config/jobs/build-drivers/build-drivers.yaml]: For testing I used the shortest running job, and set the interval to 1H so I could figure out any bugs. I just changed it back so the upload only runs on presubmits in my new commit #43931df

**Comment by jonahjon** [comment]
Test worked on https://prow.falco.org/view/s3/falco-prow-logs/logs/build-drivers-debian-periodic/1338329871952121856

**Review by leodido — APPROVED** [review]

**Pull request merged**